streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredSessionStore.java

```diff
@@ -351,10 +345,8 @@ public V fetchSession(final K key, final long earliestSessionEndTime, final long
                                                                  final K keyTo,
                                                                  final long earliestSessionEndTime,
                                                                  final long latestSessionStartTime) {
-        Objects.requireNonNull(keyFrom, "keyFrom cannot be null");
-        Objects.requireNonNull(keyTo, "keyTo cannot be null");
-        final Bytes bytesKeyFrom = keyBytes(keyFrom);
-        final Bytes bytesKeyTo = keyBytes(keyTo);
+        final Bytes bytesKeyFrom = keyFrom == null ? null : keyBytes(keyFrom);
```

Comment:
Similarly here, would it make sense to integrate that check into keyBytes? I think there are similar cases in other stores.

Reply:
Agree! Updated! Thanks.

Reply:
@guozhangwang do you want to take a quick look as well?